packages/system/src/Name/Components/SupplierSearchInput/SupplierSearchInput.tsx

```diff
@@ -1,11 +1,33 @@
 import React, { FC } from 'react';
-import { Autocomplete, useBufferState } from '@openmsupply-client/common';
-import { useSuppliers } from '../../api';
 import {
-  basicFilterOptions,
-  NameSearchInputProps,
-  simpleNameOptionRenderer,
-} from '../../utils';
+  DefaultAutocompleteItemOption,
+  AutocompleteOptionRenderer,
+  Typography,
+  Autocomplete,
+  useBufferState,
+} from '@openmsupply-client/common';
+import { useSuppliers } from '../../api';
+import { basicFilterOptions, NameSearchInputProps } from '../../utils';
+import { NameRowFragment } from '../../api';
+
+const optionRenderer: AutocompleteOptionRenderer<NameRowFragment> = (
+  props,
+  item
+) => (
+  <DefaultAutocompleteItemOption {...props}>
+    <Typography
+      sx={{
+        marginInlineEnd: '10px',
+        fontWeight: 'bold',
+        width: 75,
+        color: item.store ? 'red' : 'inherit',
```

Comment:
Made the code red for an internal supplier / store like msupply does. Open to suggestions

Reply:
I still see red as a warning indication and it's kind the reverse here.
How about a little store icon?

<img width="572" alt="Screen Shot 2022-03-14 at 9 16 29 AM" src="https://user-images.githubusercontent.com/9192912/158077727-ac41e5db-ea2d-4304-8f50-f0b32454b8e0.png">

or in the attractive colour known as 'secondary'?

<img width="427" alt="Screen Shot 2022-03-14 at 9 15 37 AM" src="https://user-images.githubusercontent.com/9192912/158077747-76c10899-1482-4da0-8494-187051ad05ab.png">

Reply:
Oo  nice idea. For the supplier list with both internal and external - no icon or an alternative?

Reply:
I was thinking no icon - highlights the internal more that way. Just a blank space the right width
Started with an 'external supplier' icon ( like the 'external link' one ) but thought it was better to highlight internal.

Side question - is there a way to get a list of all suppliers? The API is only giving me internal and I don't remember where that discussion ended up

Reply:
Committed the suggestion to 09e71ba7 in case it is helpful.

Reply:
> Side question - is there a way to get a list of all suppliers? The API is only giving me internal and I don't remember where that discussion ended up


Just adding the filter `isSupplier: true` should be enough. 

Were you checking by using the list of suppliers when making a request requisition? If so, that is using internal suppliers only. Inbound should be using both

This query is giving me all suppliers on the demo server, internal and external

```
query MyQuery {
  names(storeId: "D77F67339BF8400886D009178F4962E1", filter: {isSupplier: true}) {
    ... on NameConnector {
      nodes {
        isSupplier
        isCustomer
        id
        store {
          id
        }
      }
    }
  }
}

```


> Committed the suggestion to https://github.com/openmsupply/openmsupply-client/commit/09e71ba78be5861ef34c0f12ce84cdd53cb7ed04 in case it is helpful.

<img src='https://media0.giphy.com/media/IcGkqdUmYLFGE/giphy.gif?cid=5a38a5a252a3kjcwqtndlmyih2e5mttrqpppt8znfiqjzx86&rid=giphy.gif&ct=g'/>

Reply:
Nice.. much better than the red.

![image](https://user-images.githubusercontent.com/35858975/158083441-fad9c46b-a71f-4efc-bf91-861ea69ee38f.png)

![image](https://user-images.githubusercontent.com/35858975/158083462-ce4b6b36-5060-4424-8088-b8b4f32b668e.png)

Using `Tamaki` on the demo server